src/main/groovy/com/blackducksoftware/integration/hub/detect/bomtool/pear/PearDependencyFinder.groovy

```diff
@@ -96,15 +96,34 @@ class PearDependencyFinder {
     }
 
     private Set<DependencyNode> createPearDependencyNodeFromList(String list, List<String> dependencyNames) {
+        Set<DependencyNode> parsedDependencies = []
+        String[] channelList = list.split('=')
+        channelList -= ''
```

Comment:
I think for now, we prefer removeAll or retainAll: the '-=' operator for collections is just too terse.